kotest-assertions/kotest-assertions-shared/src/commonMain/kotlin/io/kotest/utils/CondensedIndexes.kt

```diff
@@ -0,0 +1,42 @@
+package io.kotest.utils
+
+class CondensedIndexes {
+}
+
+fun Iterable<Int>.condenseToRanges(minRangeSize: Int): Sequence<IndexesToPrint> {
```

Comment:
Should probably make this function internal

Reply:
fixed TY